packages/udaru-hapi-server/README.md

```diff
@@ -1,31 +1,326 @@
-# Udaru Server
+# Udaru
 [![npm][npm-badge]][npm-url]
 [![travis][travis-badge]][travis-url]
 [![coveralls][coveralls-badge]][coveralls-url]
 [![snyk][snyk-badge]][snyk-url]
 
-![Udaru](./docs/logo.jpg)
+<img src="https://github.com/nearform/udaru/raw/master/docs/logo.jpg">
 Udaru is a Policy Based Access Control (PBAC) authorization module. It supports Organizations, Teams and User entities that are used to build the access model. The policies attached to these entities define the 'Actions' that can be performed by an entity on various 'Resources'.
 
 See the Udaru [website](https://nearform.github.io/udaru/) for complete documentation on Udaru.
 
-## Install
-To install via npm:
+## Usage
 
+Udaru has three modules:
+
+1) `udaru-core` - the core udaru library
+
+2) `udaru-hapi-plugin` - a plugin for Hapi that exposes a HTTP interface to `udaru-core`
+
+3) `udaru-hapi-server` - a standalone server that instantiates `udaru-hapi-plugin`
+
+### Udaru Core module
+
+```bash
+$ npm install @nearform/udaru-core
+```
+
+```js
+const buildUdaru = require('@nearform/udaru-core')(dbPool, config)
+```
+
+Both the dbPool and config are optional and are initialised with default values if not provided, see below for details on both.
+
+
+### Hapi plugin
+
+```bash
+$ npm install @nearform/udaru-hapi-plugin
+```
+
+```js
+const Hapi = require('hapi')
+const UdaruPlugin = require('@nearform/udaru-hapi-plugin')
+
+...
+
+const server = new Hapi.server()
+server.register({register: UdaruPlugin})
+```
+
+### Stand alone server
+```
+$ npx udaru
```

Comment:
is npx correct?

Reply:
yes it is (see https://medium.com/@maybekatz/introducing-npx-an-npm-package-runner-55f7d4bd282b)
This is not tested though but the intent is to be able to run it via npx